.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "tabWidth": 2
```

Comment:
I've added this as sometimes vscode does not use prettier formatter if config file is missing